larwirecell/Modules/Geometry/CTreeGeometry_module.cc

```diff
@@ -63,7 +63,7 @@
 
 using namespace std;
 
-namespace ICARUS{
+namespace icarus {
```

Comment:
Recommend making this namespace entirely anonymous:

```diff
- namespace ICARUS {
+ namespace {
```

Reply:
I have no problem (and no stakes) with the suggested resolution.